src/backend/debugging.md

```diff
@@ -56,6 +56,17 @@ different stages during compilation, which is sometimes useful. One just needs
 to convert the bitcode files to `.ll` files using `llvm-dis` which should be in
 the target local compilation of rustc.
 
+If you are seeing incorrect behavior due to an optimization pass, a very handy
+LLVM option is `-opt-bisect-limit`, which takes an integer denoting the index
+value of the highest pass to run.  Index values for taken passes are stable
+from run to run; by coupling this with software that automates bisecting the
```

Comment:
Should we mention software that can be used for this?

Reply:
Sorry -- I missed this comment!  Yeah, the "software that automates bisecting" is really a euphemism for: "the gnarly script that you, dear reader, are about to write." ;)  Happy to spell that out, if with slightly less punchy language..